docs/api_core.rst

```diff
@@ -1970,6 +1970,18 @@ The following annotations can be specified using the variable-length
    ``Shape(2) * 1.5 == 3.0``.) It is unspecified whether operations on
    mixed enum types (such as ``Shape.Circle + Color.Red``) are
    permissible.
+   Changes the resulting Python enumeration type to be based on `enum.IntEnum`.
+
+   .. cpp:struct:: flag_enum
```

Comment:
Should this be changed back to `is_flag` to be more in line with the other flags? That came up already in the previous PR.

Reply:
Yeah, I prefer that name.